.buildkite/pipeline.yml

```diff
@@ -47,6 +47,8 @@ steps:
       queue: "${BUILDKITE_AGENT_META_DATA_QUEUE:-default}"
 
   - label: ':hammer: tests'
+    FIRECRACKER_GO_SDK_INIT_TIMEOUT_SECONDS: 3
+    FIRECRACKER_GO_SDK_REQUEST_TIMEOUT_MILLISECONDS: 500
```

Comment:
I'd actually prefer that we not set these here, even if we're just explicitly setting the default value. Using the default value during our automated testing is the right thing to do, but explicitly setting the values introduces an opportunity for configuration drift.

Reply:
Hmm, then how do you suggest to set it?

Reply:
I mean how should regular users set this env's?

Reply:
What you're doing is exactly right, and I think this is how users should set these variables when they do. I just don't think we should hardcode these settings here. When we run the tests in buildkite, we should rely on the defaults from the go code. If somebody needs to debug something and tweak these values, they can set environment variables, either here in the buildkite config (temporarily, during debugging) or in their runtime environment.

Reply:
Then we definitely should point it in some kind of documentation or somewhere else, since anyone except us, don't know a thing about those env's.

Reply:
Yup, maybe add something to [HACKING.md](/firecracker-microvm/firecracker-go-sdk/HACKING.md)?